docs/developer_guide/internal-workflow.md

```diff
@@ -0,0 +1,112 @@
+# Internal Workflow at MedModels
+
+## IDE Config: VSCode
+
+For a smooth development experience in VS Code, we recommend installing the following extensions. You can install them directly within VS Code using the provided `extensions.json` file. Here's how to do it:
+
+1. Open the VS Code extensions view (**Go** > **Extensions** or use the shortcut <kbd>Ctrl+Shift+X</kbd> on Windows/Linux or <kbd>Cmd+Shift+X</kbd> on macOS).
+2. Click the **Import Extensions** icon (three dots stacked diagonally) in the extensions view.
+3. Select the provided `extensions.json` file from your local project directory.
+
+This will import the list of recommended extensions and prompt you to install them.
+
+**Recommended Extensions:**
+
+The `.vscode/extensions.json` file includes the following extensions to enhance your development experience:
+
+```json
+{
+  "recommendations": [
+    "ms-python.python",
+    "charliermarsh.ruff",
+    "tamasfe.even-better-toml",
+    "ms-vscode-remote.remote-containers",
+    "EditorConfig.EditorConfig",
+    "eamodio.gitlens",
+    "ms-vsliveshare.vsliveshare",
+    "ms-vscode.makefile-tools",
+    "rust-lang.rust-analyzer",
+    "njpwerner.autodocstring",
+    "usernamehw.errorlens"
+  ]
+}
+```
+
+**VS Code Settings:**
+
+Once you've installed the recommended extensions, consider adding the following settings to your VS Code settings file (`.vscode/settings.json`) to optimize your development workflow. These settings are provided below for your reference:
+
+```json
+{
+  "editor.formatOnSave": true,
+  "editor.formatOnPaste": true,
+  "editor.autoIndent": "advanced",
+  "files.associations": {
+    "setup.cfg": "ini",
+    "rust-toolchain": "toml"
+  },
+  "editor.rulers": [88],
+  "[rust]": {
+    "editor.rulers": [100]
+  },
+  "files.trimTrailingWhitespace": true,
+  "files.insertFinalNewline": true,
+  "python.terminal.activateEnvironment": true,
+  "python.defaultInterpreterPath": "${workspaceFolder}/.venv/bin/python",
+  "python.terminal.activateEnvInCurrentTerminal": true,
+  "autoDocstring.docstringFormat": "google",
+  "python.analysis.typeCheckingMode": "strict",
+  "[python]": {
+    "editor.defaultFormatter": "charliermarsh.ruff"
+  }
+}
+```
+
+## Issue Management
```

Comment:
```suggestion
## Issues Management
```

Reply:
singular is correct here imo (and apparently chatgpts opinion). Similar to how `Issue Types` is `Issue Types` and not `Issues Types`